constraints.pro

```diff
@@ -350,13 +350,6 @@ gen_enforced_field(WorkspaceCwd, 'publishConfig.registry', 'https://registry.npm
 gen_enforced_field(WorkspaceCwd, 'publishConfig', null) :-
   workspace_field(WorkspaceCwd, 'private', true).
 
-% nonce-tracker has an unlisted dependency on babel-runtime (via `ethjs-query`), so that package
```

Comment:
This is great!

Can we remove `babel-runtime` as a dev dependency of `transaction-controller`?

Reply:
Okay, then do we still want to keep this constraint in the meantime?

Reply:
Right :)